src/components/CommentsSection.css

```diff
@@ -0,0 +1,56 @@
+.comments-section {
+    width: 80%;
+    max-width: 1200px;
+    background-color: #fff;
+    padding: 1.25rem;
+    border-radius: 10px;
+    margin-top: 1.25rem;
+    margin-bottom: 1.25rem;
+    gap: 1rem;
+    box-shadow: 0px 4px 8px rgba(0, 0, 0, 0.1);
+}
+
+.delete-icon {
+    color: #555;
+    /* Customize icon color */
+    cursor: pointer;
+    margin-left: 10px;
+    font-size: 1.2rem;
+    /* Adjust icon size if needed */
+}
+
+.delete-icon:hover {
+    color: #B22222;
+    /* Hover color for delete icon */
+}
+
+.comment-input {
+    display: flex;
+    gap: 0.75rem;
+    margin-top: 0.75rem;
+}
+
+.comment-input input[type="text"] {
+    flex-grow: 1;
+    padding: 0.75rem;
+    border: 1px solid #ddd;
+    border-radius: 20px;
+    font-size: 1rem;
```

Comment:
Please pick either rem or px. In case you don't know how to convert, 1rem = 16px